File: test/declare_class.java
class Test {
    public static void main(String[] arg) {
        System.out.println(!true);
    }
}

class Test2 {
    int x;
    int y;

    public int sum(int x, int y) {
        return x + y;
    }
}

class Test3 extends Test2 {
    public int sum(int x, int y) {
        return y + x;
    }
}

File: test/declare_class.py
import sys
from typing import List


def main():
    arg = sys.argv[1:]
    print((not True))


class Test2:
    x = None
    y = None

    def sum(x: int, y: int) -> int:
        return (x + y)


class Test3(Test2):

    def sum(x: int, y: int) -> int:
        return (y + x)


if __name__ == "__main__":
	main()
